src/server/views/partials/admin-teamoverview.handlebars

```diff
@@ -29,10 +29,11 @@
         <table class="table table-striped">
             <thead>
             <tr>
-                <th>Team-Name</th>
-                <th>Mitglieder</th>
+                <th>Status</th>
+                <th style="width: 150px">Team</th>
+                <th style="width: 200px">Participants</th>
                 <th>
-                  Letzer Post
+                  Last Post
```

Comment:
No Translations needed for this?

Reply:
Got lazy. And this is only for admins anyway, where nothing is translated